portal/static/js/src/modules/TnthAjax.js

```diff
@@ -353,12 +403,17 @@ export default { /*global $ */
                 } else {
                     $(".delete-consent-error").html(i18next.t("Server error occurred removing consent."));
                 }
+                callback();
             });
         });
     },
     withdrawConsent: function(userId, orgId, params, callback) {
         callback = callback || function() {};
         params = params || {};
+        if (!params.research_study_id) {
+            params.research_study_id = 0;
```

Comment:
Might want to use a constant or something for the default research study ID (`EPROMS_SUBSTUDY_ID`?), so it's easier to find/replace later if necessary

Reply:
thanks ivan, fixed